### ISSUE TITLE
Translation: ES - Spanish

### ISSUE THREAD

**Issue by SantiagoGnzlz** [ncase/remember#10]
I'm going to start translating. To anyone who wants to help, feel free to do so.

**Comment by SantiagoGnzlz** [comment]
I'm having trouble displaying the cards when I preview my alternate html file. Should I add it to translations.txt? Does that enable the javascript boxes?

**Comment by ncase** [comment]
Huh, strange. :/ What errors (if any) does the JavaScript console say?

Are you using a localhost server, or previewing the .html file by itself? On Chrome (and maybe other browsers), if you view from file:// it _does not allow iframes to communicate with each other_. The solution I use is http-server, a no-config localhost server: https://www.npmjs.com/package/http-server Hope that resolves the problem, and thanks again!

**Comment by SantiagoGnzlz** [comment]
Yeah, I think that's it, the preview WebStorm gives me seems to be just the file opened in the explorer. I'll configure the localhost server and see if it fixes it. Thanks!

**Comment by fititnt** [comment]
Nice. Someone is translating to Spanish.

 @SantiagoGnzlz if you want some help with the setup part, you can ping me (rocha @ ieee.org). I'm online at this moment, so we can even share screen on Google Hangout (or other service) instead of via e-mail. Is possible to setup something like this https://github.com/fititnt/remember/pull/1 and make easier for other people review (or even to yourself).

**Comment by SantiagoGnzlz** [comment]
Thanks for the offer! I was able to set up the localhost server in WebStorm, but I'll try that method too later down the line so that other people can check it too.

**Comment by SantiagoGnzlz** [comment]
@ncase I've finished the translation and created a pull request (#12 )
